mitodl/micromasters
# Added audit table for Order, Line

#### What are the relevant tickets?
Fixes #1421 

#### What's this PR do?
Adds an audit table for `Order` and code to log changes to `Order` in the checkout and order fulfillment APIs.

#### How should this be manually tested?
Create an order by clicking the 'Buy now' button. Verify that there is a change in `OrderAudit` which also includes the data for the `Line`.

Then feed the purchase email to the order fulfillment API. (This doesn't have to succeed, failure will work too.) Verify that there is a new change in `OrderAudit` reflecting that change of status.
